src/EnergyPlus/General.hh

```diff
@@ -157,6 +157,27 @@ namespace General {
                    Real64 &XX_1                // Hign bound obtained with maximum number of allowed iterations
     );
 
+    constexpr Real64 InterpGeneral(Real64 const Lower,
+                                Real64 const Upper,
+                                Real64 const InterpFac
+    )
+    {
+        return Lower + InterpFac * (Upper - Lower);
+    };
+
```

Comment:
I'm sure declaring this function `constexpr` vs. just `inline` doesn't hurt anything, but I doubt any of the arguments will actually be `constexpr` and enable any compile-time reduction. Same for `InterpProfSlat`.